azure-template/sdk_packaging.toml

```diff
@@ -0,0 +1,7 @@
+[packaging]
+auto_update = false
+package_name = "azure-template"
+package_pprint_name = "Template Package"
+package_doc_id = "<docId>"
```

Comment:
If the template is used, this will trigger incorrect url with `<docId>` inside. Prefer a comment.

Reply:
Got it. Updating